### ISSUE TITLE
CEP com mensagem de erro

### ISSUE THREAD

**Issue by Andrefcp** [pedro-teixeira/correios#272]
Ola. Não meu site, quando pesquiso alguns CEPs aparece a seguinte mensagem junto com a cotação do SEDEX:
"Erro ao calcular tarifa no SGPB. CEP de origem não pode postar para o CEP de destino informado(-1)."

O que seria essa mensagem?
Esse CEP tem restrição dos Correios. Veja: 08215-430

**Comment by rafaelpatro** [comment]
@Andrefcp, o mesmo problema foi reportado há alguns dias na PR #254.

O que ocorre é que um dos serviços cotados (provavelmente PAC) retornou um erro -888. Isto é uma restrição dos Correios, que não postará mais PAC entre os CEPs em questão.
Por alguma falha no módulo, o erro está sendo exibido junto às outras cotações válidas.
Ainda não foi solucionado.

**Comment by Andrefcp** [comment]
Mas outros CEPs para a mesma cidade não ocorre esse erro. Aparece apenas a cotação do SEDEX sem essa mensagem....
 Será que essa mensagem não seria para CEP com restrição dos Correios alegando que tem prazo diferenciado.

**Comment by rafaelpatro** [comment]
Sim, exatamente. O erro aparece apenas para CEPs com restrição de entrega, e que estão indisponíveis para entrega via PAC.

**Comment by rafaelpatro** [comment]
Enquanto ainda não aparece uma solução, você pode contornar o problema com essa [dica aqui](https://github.com/pedro-teixeira/correios/issues/254#issuecomment-306874679).
